clear/main.py

```diff
@@ -216,6 +216,17 @@ def clear_wm_pg_shared():
     command = 'watermarkstat -c -t pg_shared'
     run_command(command)
 
+@priority_group.group()
+def drop():
+    """Clear priority group dropped packets stats"""
+    pass
+
+@drop.command('counters')
+def clear_pg_counters():
+    """Clear PG dropped packets stats counter """
```

Comment:
why do we use sometimes PG and some time priority group? lets use one terminology in both UI and code
may i suggest to remove the stats? it has nothing to do with stats but these are counters.